src/objects/LOD.js

```diff
@@ -74,6 +74,23 @@ class LOD extends Object3D {
 
 	}
 
+	removeLevel( level ) {
```

Comment:
Now that I think about it: Can't you just directly remove the level from the `levels` array?

Do you mind providing a code snippet that shows how you get/create the `level` parameter before removing it from the LOD?